### PR TITLE
Add a test that verifies that empty income is invalid data from UI

### PR DESCRIPTION
**Pull request by androa**

### CODE REVIEW

File: src/test/kotlin/no/nav/dagpenger/inntekt/v1/UklassifisertInntektApiTest.kt [hunk 1]
```diff
@@ -258,6 +267,55 @@ class UklassifisertInntektApiTest {
         }
     }
 
+    @Test
+    fun `Post uklassifisert inntekt med feil redigert should return 200 ok`() = testApp {
```
Comment (tonjemjovik) [inline src/test/kotlin/no/nav/dagpenger/inntekt/v1/UklassifisertInntektApiTest.kt]: 200 ok?

Comment (androa) [reply src/test/kotlin/no/nav/dagpenger/inntekt/v1/UklassifisertInntektApiTest.kt]: Der har du rett i :D 

Se nå da, dette er gøy:

```suggestion
    fun `Post uklassifisert inntekt med feil redigert should return 400 ok`() = testApp {
```
